styles/styles.css

```diff
@@ -0,0 +1,768 @@
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+.imgproject__photos_row {
+  margin-bottom: 30px;
+  display: flex;
+  flex-direction: row;
+}
+.imgproject__photos_column {
+  margin-right: 30px;
+  justify-content: space-between;
+  display: flex;
+  flex-direction: column;
+}
+
+.imgproject__photos_link {
+  display: flex;
+  position: relative;
+}
+.imgproject__photos_link__text {
+  display: flex;
+  height: 92px;
+  width: 495px;
+  bottom: 24px;
+  left: 19px;
+  position: absolute;
+  background-color: white;
+}
+.imgproject__photos_link__text p {
+  font-size: 18px;
+  text-align: start;
+  margin-left: 30px;
+  margin-top: 25px;
+}
+.imgproject__photos_link__text p span {
+  font-size: 14px;
+  color: #9ca3af;
+}
+.imgproject__photos_link__text p {
+  line-height: 20px;
+  font-weight: 400;
+}
+.imgproject__photos_link__text img {
+  display: flex;
+  align-items: center;
+  position: absolute;
+  display: inline-block;
+  right: 30px;
+}
+.imgproject__photos_link__text__link {
+  display: flex;
+  align-items: center;
+}
+.imgproject__photos_link__text__link a {
+  display: flex;
+  align-items: center;
+}
+
+.planning {
+  margin: 0 328px;
+  margin-top: 150px;
+  justify-content: space-between;
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  font-weight: 600;
+}
+.planning__block__text_bold {
+  font-size: 30px;
+}
+.planning__block__text {
+  display: block;
+  font-weight: 400;
+  margin-top: 24px;
+  font-size: 18px;
+  color: #9ca3af;
+  line-height: 26px;
+}
+.planning__block__img {
+  margin-bottom: 5px;
+}
+.careers {
+  margin-top: 115px;
+}
+.careers__logo {
+  font-weight: 600;
+  text-align: center;
+  font-size: 48px;
+}
+.careers div {
+  display: flex;
+  justify-content: space-between;
+
+  margin-top: 58px;
+  margin-left: 152px;
+}
+.careers__company,
+.careers__position,
+.careers__year {
+  flex-direction: column;
+}
+.careers__text {
+  margin: 30px;
+  text-align: left;
+  font-size: 16px;
+  color: #6b7280;
+  margin-bottom: 54px;
+}
+.careers__company p span,
+.careers__position p span,
+.careers__year p span {
+  font-weight: 600;
+}
+.careers__position {
+  margin-right: -18px;
+}
+.careers__year {
+  margin-right: 433px;
+}
+footer {
+  margin-top: 100px;
+  background-color: #fef2f2;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.footer__text {
+  text-align: center;
+  font-size: 48px;
+  margin-bottom: 20px;
+  font-weight: 500;
+  margin-top: 104px;
+}
+.footer__span {
+  text-align: center;
+  font-size: 18px;
+  color: #9ca3af;
+  margin-bottom: 50px;
+  line-height: 24px;
+}
+footer img {
+  margin-bottom: 100px;
+}
+.footer__contacts {
+  display: flex;
+  height: 100px;
+  width: 60%;
+  justify-content: space-around;
+  margin-top: 53px;
+  margin-bottom: 45px;
+}
+.footer__contacts a {
+  height: 35px;
+}
+@media only screen and (min-width: 769px) and (max-width: 1280px) {
```

Comment:
The idea is to accumulate styles when you get down so you don't need to to write the same styles every breakpoint. To make it you don't need restrict it from both sides (max-width and min-width), use max-width only:
default styles -> (max-width: 1280px) (and less) -> (max-width: 768px) (includes styles from previous 1280px and less) -> (max-width: 550px) (and so on) -> and less.

Look at this to get an idea
https://habr.com/ru/post/269419/

Don't need to change it this time but good to know for the future.